### DISCUSSION TITLE
Not appearing on "Currently playing" section of osu!lazer

### DISCUSSION THREAD

**Discussion by Aikiyaa** [ppy/osu#19734] [Q&A]
### Type

Game behaviour

### Bug description

While trying to have my friend spectate me solo in osu lazer I see that I don't appear on the "currently playing" tab even though I am online and playing the game. Any reason as to why this is?

### Version

2022-810.2-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/9330768/performance.log)
[database.log](https://github.com/ppy/osu/files/9330774/database.log)
[network.log](https://github.com/ppy/osu/files/9330776/network.log)
[runtime.log](https://github.com/ppy/osu/files/9330777/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9330778/updater.log)

**Comment by peppy** [comment]
Nothing in your logs indicates an issue.

Note that to show up, you should be playing a ranked beatmap, and you will only show up while actually playing.
